circus-train-s3-mapreduce-cp/src/main/java/com/hotels/bdp/circustrain/s3mapreducecp/S3MapReduceCpOptions.java

```diff
@@ -442,28 +443,53 @@ public void setCannedAcl(String cannedAcl) {
 
   @Override
   public String toString() {
-    return "S3MapReduceCpOptions{" +
-            "help=" + help +
-            ", async=" + async +
-            ", sources=" + sources +
-            ", target=" + target +
-            ", credentialsProvider=" + credentialsProvider +
-            ", multipartUploadPartSize=" + multipartUploadPartSize +
-            ", s3ServerSideEncryption=" + s3ServerSideEncryption +
-            ", storageClass='" + storageClass + '\'' +
-            ", maxBandwidth=" + maxBandwidth +
-            ", numberOfUploadWorkers=" + numberOfUploadWorkers +
-            ", multipartUploadThreshold=" + multipartUploadThreshold +
-            ", maxMaps=" + maxMaps +
-            ", copyStrategy='" + copyStrategy + '\'' +
-            ", logPath=" + logPath +
-            ", region='" + region + '\'' +
-            ", ignoreFailures=" + ignoreFailures +
-            ", s3EndpointUri=" + s3EndpointUri +
-            ", uploadRetryCount=" + uploadRetryCount +
-            ", uploadRetryDelayMs=" + uploadRetryDelayMs +
-            ", uploadBufferSize=" + uploadBufferSize +
-            ", cannedAcl='" + cannedAcl + '\'' +
-            '}';
+    return "S3MapReduceCpOptions{"
```

Comment:
eh? this is horrible

Reply:
It is! It seems that eclipse auto-formats it like this, but I can undo this